core/templates/pages/exploration-editor-page/feedback-tab/feedback-tab.directive.html

```diff
@@ -83,9 +83,9 @@ <h2>Exploration Feedback</h2>
 
               <div class="row">
                 <div class="col-lg-12 col-md-12 col-sm-12">
-                  <div style="white-space: pre-wrap; word-break: break-word" class="protractor-test-exploration-feedback"><[m.text]></div>
+                  <div class="oppia-exploration-feedback protractor-test-exploration-feedback"><[m.text]></div>
```

Comment:
This seems to be incorrectly named. Add the element you are refering to in the class like here it would need to end with -text.